core/src/com/unciv/ui/components/extensions/Scene2dExtensions.kt

```diff
@@ -293,6 +293,28 @@ fun getCloseButton(
     return closeButton
 }
 
+/**
+ * Adds a white-circled (x) close button to [parent], positioned to the top right,
+ * slightly shifted outwards.
+ */
+fun addRoundCloseButton(
+    parent: Group? = null,
+    size: Float = 30f,
+    iconSize: Float = size - 15f,
+    circleColor: Color = Color.CLEAR,
+    overColor: Color = Color.RED,
+    action: () -> Unit
```

Comment:
This suffers from Trog Disease, which is adding overrides that are not actually used when they can easily be added in the future when needed
Please minimize to only what we currently need, and push the specifics into the function :)

Reply:
e.g. parent stays, but is only Group (never null)
All the rest are hardcoded in the function

Reply:
Thank you for the feedback, fixed :+1: